File: evaluation-corpus/degenerate/java/dg_quinn.java
public class PointQuinn {
    private final int row;
    private final int col;
    public PointQuinn(int row, int col) { this.row = row; this.col = col; }
    public int getRow() { return row; }
    public int getCol() { return col; }
    public String toString() { return "(" + row + ", " + col + ")"; }
}

File: evaluation-corpus/degenerate/python/dg_quinn.py
class Point:
    def __init__(self, row, col):
        self.row = row
        self.col = col

    def get_row(self):
        return self.row

    def get_col(self):
        return self.col

    def __str__(self):
        return "(%d, %d)" % (self.row, self.col)
